src/redux/validate.ts

```diff
@@ -49,6 +54,8 @@ ajv.getSchema('#/definitions/IncomeW2')
 ajv.getSchema('#/definitions/EditW2Action')
 ajv.getSchema('#/definitions/EstimatedTaxPayments')
 ajv.getSchema('#/definitions/EditEstimatedTaxesAction')
+ajv.getSchema('#/definitions/HealthSavingsAccounts')
```

Comment:
If you fix this to `ajv.getSchema('#/definitions/HealthSavingsAccount')`, the validate error seems to go away. (drop the `s`).

Reply:
Thank you! I was starting at this for more than an hour.